src/EditorFeatures/CSharp/NavigationBar/CSharpNavigationBarItemService.cs

```diff
@@ -317,6 +232,8 @@ private static void AddEnumMemberSpan(ISymbol symbol, SyntaxTree tree, List<Text
         protected internal override VirtualTreePoint? GetSymbolItemNavigationPoint(Document document, NavigationBarSymbolItem item, CancellationToken cancellationToken)
         {
             var compilation = document.Project.GetCompilationAsync(cancellationToken).WaitAndGetResult(cancellationToken);
+            // If we're being called back, there must have been a compilation to resolve the item in the first place
+            Debug.Assert(compilation is not null);
```

Comment:
```suggestion
            var compilation = document.Project.GetRequiredCompilationAsync(cancellationToken).WaitAndGetResult(cancellationToken);
```

Same thing as above.  all C#/VB documents have compilations.

Reply:
this comment still applies.